code/modules/surgery/embalming.dm

```diff
@@ -11,8 +11,7 @@
 
 /datum/surgery_step/embalming
 	name = "embalming body"
-	implements = list(/obj/item/reagent_containers/syringe = 100, /obj/item/pen = 30) 
-	time = 10
+	implements = list(TOOL_HEMOSTAT = 100, TOOL_SCREWDRIVER = 35, /obj/item/pen = 0)
```

Comment:
Remove the /obj/item/pen entry, you can already fail surgery steps on purpose by starting them with disarm intent on.